tests/suites/deploy/deploy_bundles.sh

```diff
@@ -158,7 +158,7 @@ run_deploy_charmhub_bundle() {
 	juju deploy "${bundle}"
 
 	wait_for "juju-qa-test" "$(charm_channel "juju-qa-test" "2.0/stable")"
-	wait_for "juju-qa-test-focal" "$(charm_channel "juju-qa-test-focal" "candidate")"
+	wait_for "juju-qa-test-focal" "$(charm_channel "juju-qa-test-focal" "latest/candidate")"
```

Comment:
Longer term, we should consider how to make 'charm_channel' more friendly for when it is failing. We had *a* valid charm channel, it just wasn't the one we thought it should be, and it made it hard to understand why the test was failing.